dask/array/tests/test_fft.py

```diff
@@ -18,6 +18,20 @@ def key(k):
     return sorted(a.dask, key=key) == sorted(b.dask, key=key)
 
 
+@pytest.fixture(
+    params=[
+        "fft",
+        "ifft",
+        "rfft",
+        "irfft",
+        "hfft",
+        "ihfft",
+    ]
+)
+def funcname(request):
+    return request.param
```

Comment:
FWIW I actually disagree with @jreback here.  I think that it's way simpler to just set

```python
funcname = ['fft', 'ifft', 'rfft', ...]
```

In this case this has exactly the same effect and uses less technology.